package.json

```diff
@@ -24,18 +24,27 @@
     "build": "lerna exec yarn build --stream",
     "clean": "polkadot-dev-clean-build",
     "electron": "cd packages/electron-app && yarn electron",
+    "fetch-substrate": "cd ./scripts && node fetchLatestSubstrate.js",
     "lint": "tsc --noEmit && tslint --project .",
     "package": "cd packages/electron-app && yarn package",
     "prebuild": "yarn clean",
     "preelectron": "yarn build",
     "prepackage": "yarn build",
     "prestart": "./scripts/prestart.sh",
+    "postinstall": "yarn fetch-substrate",
     "start": "lerna exec yarn start --parallel",
     "storybook": "cd packages/ui-components && yarn storybook",
     "storybook-build": "cd packages/ui-components && yarn build-storybook",
     "test": "CI=true lerna run test --parallel"
   },
   "devDependencies": {
     "@polkadot/dev": "^0.25.13"
+  },
+  "dependencies": {
+    "fs": "^0.0.1-security",
+    "node-fetch": "^2.5.0",
+    "path": "^0.12.7",
+    "semver": "^6.0.0",
+    "util": "^0.12.0"
```

Comment:
util, path and fs are already inside node.js, no need for external dependencies (i'm not even sure which `util` is taken when you do `require('util')` in this case)